tests/pytests/functional/modules/test_cmdmod.py

```diff
@@ -0,0 +1,123 @@
+import pytest
+import salt.modules.cmdmod as cmdmod
+import salt.utils.path
+
+pytestmark = [
+    pytest.mark.windows_whitelisted
```

Comment:
```suggestion
    pytest.mark.windows_whitelisted,
    pytest.mark.skip_unless_on_windows,
```

Reply:
Done. Though if we add non Windows tests here in the future, we'll have to move them back.